lib/router/routes.dart

```diff
@@ -32,9 +32,11 @@ final routemaster = RoutemasterDelegate(
       '/home/recipes': (_) => MaterialPage(child: Scaffold(appBar: AppBar())),
       '/home/dictionary': (_) => MaterialPage(child: Scaffold(appBar: AppBar())),
       '/home/letters': (_) => MaterialPage(child: Scaffold(appBar: AppBar())),
-      '/setting': (_) => MaterialPage(child: Scaffold(appBar: AppBar())),
-      '/about_us': (_) => MaterialPage(child: Scaffold(appBar: AppBar())),
-      '/help': (_) => MaterialPage(child: Scaffold(appBar: AppBar())),
+      '/home/setting': (_) => MaterialPage(child: Scaffold(appBar: AppBar())),
```

Comment:
ここNotionに合わせて変えておいてほしい
```suggestion
      '/home/user_setting': (_) => MaterialPage(child: Scaffold(appBar: AppBar())),
```